permiso/src/main/java/com/greysonparrelli/permiso/PermisoDialogFragment.java

```diff
@@ -17,9 +20,13 @@
 
     public static final String TAG = "PermisoDialogFragment";
 
-    private static final String KEY_TITLE = "title";
-    private static final String KEY_MESSAGE = "message";
+    private static final String KEY_TITLE       = "titleId";
```

Comment:
Personal nit: I don't like adding extra alignment spaces to get all of the ```=``` to line up.